src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -35,13 +36,12 @@ def name_to_json(names):
         namedict.append({'name': names[i].decode('utf-8')})
     return json.dumps(namedict, indent=4, sort_keys=True)
 
-class VolumeClient(object):
+
+class VolumeClient(CephfsClient):
```

Comment:
nit: VolumeClient::shutdown() should call CephfsClient::shutdown() or if VolumeClient is overriding it (note that the call order matters) then please add a comment.